packages/canary-react/src/stories/ic-date-input.stories.mdx

```diff
@@ -607,3 +608,86 @@ The date input can be cleared by setting the the value attribute to one of the f
     <ClearDateInput />
   </Story>
 </Canvas>
+
+### Playground example
+
+Go to the <IcLink href="/?path=/story/react-components-date-input--playground-example">separate page for the playground example</IcLink> to view the prop controls.
+
+export const defaultArgs = {
+  dateFormat: "DD/MM/YYYY",
+  disabled: false,
+  disableDays: [],
+  disableDaysMessage: "This day is not available",
+  disableFuture: false,
+  disableFutureMessage: "This date is in the future",
+  disablePast: false,
+  disablePastMessage: "This date is in the past",
+  helperText: "Choose a date",
+  label: "When would you like to collect your coffee?",
+  max: "",
+  min: "",
+  required: false,
+  showClearButton: true,
+  size: "default",
+  validationStatus: "",
+  validationText: "",
+  value: "",
+};
+
+<Canvas withSource="none">
+  <Story
+    args={defaultArgs}
+    argTypes={{
+      dateFormat: {
+        options: ["DD/MM/YYYY", "MM/DD/YYYY", "YYYY/MM/DD"],
+        control: {
+          type: "radio",
+        },
+      },
+      validationStatus: {
+        options: ["", "error", "success", "warning"],
+        control: {
+          type: "radio",
+        },
+      },
+      size: {
+        options: ["small", "default", "large"],
+        control: {
+          type: "radio",
+        },
+      },
+    }}
+    name="Playground example"
+  >
+    {(args) => {
+
+      const [{ value }, updateArgs] = useArgs();
+
+      return(
+       <IcDateInput
+          dateFormat={args.dateFormat}
+          disabled={args.disabled}
+          disableDays={args.disableDays}
+          disableDaysMessage={args.disableDaysMessage}
+          disableFuture={args.disableFuture}
+          disableFutureMessage={args.disableFutureMessage}
+          disablePast={args.disablePast}
+          disablePastMessage={args.disablePastMessage}
+          helperText={args.helperText}
+          label={args.label}
+          max={args.max}
+          min={args.min}
+          openAtDate={args.openAtDate}
+          required={args.required}
+          showDaysOutsideMonth={args.showDaysOutsideMonth}
+          showPickerClearButton={args.showPickerClearButton}
+          showPickerTodayButton={args.showPickerTodayButton}
+          size={args.size}
+          startOfWeek={args.startOfWeek}
+          validationStatus={args.validationStatus}
+          validationText={args.validationText}
+          value={value}
```

Comment:
think you need to add an onIcChange event that calls updateArgs with the new value, to keep the control in sync:

```
onIcCheck={(ev) => updateArgs({ value: ev.detail.value })}
```